packages/web-components/src/label/README.md

```diff
@@ -0,0 +1,127 @@
+# Label
+
+> A label represents a caption for an item in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Label Design Spec in Figma](https://www.figma.com/file/jpWO2FMBefirTyThf5Rg2P/Label?node-id=2%3A476&t=QCdofuTbXkUjMS4d-0)
+
+<br />
+
+## **Engineering Spec**
+
+Due to the cost of associating elements across the shadow DOM boundary the `fluent-label` will not have access to the element it is intended to reference. For this reason the `fluent-label` is for visual purposes only.
+
+The fluent-label has several visual font size (small, medium, large) and font weight(regular, semibold) options. The fluent-label also provides appearances for required and disabled states.
+
+### Use Case
+
+Creating a simple label element with an optional info icon and optional required state
+
+<br />
+
+## Class: `Label`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name       | Privacy | Type                           | Default     | Description                          |
+| ---------- | ------- | ------------------------------ | ----------- | ------------------------------------ |
+| `id`       | public  | `string`                       |             | Specifies the id of label            |
+| `required` | public  | `boolean`                      | `false`     | Specifies required styling for label |
+| `disabled` | public  | `boolean`                      | `false`     | Sets disabled state for label        |
+| `size`     | public  | `"small"` `"medium"` `"large"` | `"medium"`  | Specifies font size for label        |
+| `weight`   | public  | `"regular"` `"semibold"`       | `"regular"` | Specifies font weight for label      |
+
+<br />
+
+### **Methods**
+
+<br />
+
+### **Events**
+
+<br />
+
+### **Attributes**
+
+| Name       | Field    |
+| ---------- | -------- |
+| `required` | required |
+| `disabled` | disabled |
+| `size`     | size     |
+| `weight`   | weight   |
+| `id`       | id       |
+
+<br />
+
+### **Slots**
+
+| Name | Description                            |
+| ---- | -------------------------------------- |
+|      | Default slotted content for label text |
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+```ts
+`<template
+    ?disabled="${(attr: Label) => attr.disabled}"
+    ?required="${(attr: Label) => attr.required}"
+    size="${(attr: Label) => attr.size}"
+    weight="${(attr: Label) => attr.weight"}>
+    <span
+      class="label"
+      part="label"
+      id="${(attr: Label) => attr.id}">
+    <slot></slot>
+    ${(attr: Label) => attr.required
+      ? html`<span aria-hidden="true" part="asterisk" class="asterisk">*</span>`
+      : null }
+  </span>
```

Comment:
@brianchristopherbrady is the span within the shadowroot here? If so, the `id` is unique to the Shadow DOM and will not associate. Is the span necessary for any reason? The host element (what you'd be binding data to w/ the template element) is a generic just like the span.

Reply:
removed the span